snapshot/policy/retention_policy.go

```diff
@@ -160,6 +168,14 @@ func (r *RetentionPolicy) getRetentionReasons(i int, s *snapshot.Manifest, cutof
 			continue
 		}
 
+		if *r.KeepMinDays != *newOptionalInt(0) {
+			if !s.StartTime.Before(cases[6].cutoffTime) {
+				keepReasons = []string{"Inside min retention days"}
```

Comment:
Should we append the reason here instead? There may be an overlap in the reasons for which a snapshot qualifies to be retained. Since `within` is the last member of `cases` struct, this won't get clobbered right now, but what if the order of members in `cases` struct changes in the future?